hs_app_netCDF/models.py

```diff
@@ -516,7 +516,7 @@ def get_xml(self, pretty_print=True, include_format_elements=True):
         xml_string = super(NetcdfMetaData, self).get_xml(pretty_print=False)
 
         # create an etree xml object
-        RDF_ROOT = etree.fromstring(xml_string)
+        RDF_ROOT = etree.fromstring(xml_string.encode())
```

Comment:
Wondering if these are part of a bad merge as well.  I don't think you made these changes.
https://github.com/hydroshare/hydroshare/pull/3792/files#diff-59e1dbfb48e4736f548d8c538638b652R1897
and 
https://github.com/hydroshare/hydroshare/pull/3792/files#diff-a5b057005a4b38ef14ce9dc768e306b5R89